src/Network/Session.php

```diff
@@ -347,6 +347,26 @@ public function start()
         return $this->_started;
     }
 
+    /**
+     * Write datas and close the session
+     * @return bool True if session was started
+     * @throws \RuntimeException if the session was already started
+     */
+    public function close()
+    {
+        if (!$this->_started) {
+            return true;
+        }
+
+        if (!session_write_close()) {
+            throw new RuntimeException('Could not close the session');
```

Comment:
Is there any more information we can give about the failure?

Reply:
session_write_close only return true or false, so we can't "improve" the failure message.

Reply:
Ok.